specifyweb/specify/migration_utils/sp7_schemaconfig.py

```diff
@@ -79,4 +79,89 @@
+        ('yesno1', 'YesNo1', 'YesNo1'),
+        ('isPrimary', 'Is Primary?', 'The Collection Object designated as the most significant item in a Consolidated COG. A CO child must be set as “primary” when using a “Consolidated” COG.'),
+        ('childCo', 'Child Collection Object', 'Child Collection Object'),
+        ('childCog', 'Child Collection Object Group', 'Child Collection Object Group'),
+        ('ParentCog', 'Parent', 'Parent Collection Object Group'),
+        ('yesno3', 'YesNo3', 'YesNo3'),
+    ],
+    'collectionobjectgrouptype' : [
+        ('cogTypeId', 'Collection Object Group Type ID', 'Collection Object Group Type ID'),
+        ('yesno3', 'YesNo3', 'YesNo3'),
+    ],
+    'collectionobjecttype': [
+        ('collectionObjectTypeId', 'Collection Object Type ID', 'Collection Object Type ID'),
+        ('taxonTreeDef', 'Taxon Tree', 'The Taxon Tree associated with this Collection Object Type'),
+    ],
+    'absoluteage': [
+        ('yesno2', 'YesNo2', 'YesNo2'),
+    ],
+    'paleocontext': [
+        ('yesno1', 'YesNo1', 'YesNo1'),
+        ('yesno3', 'YesNo3', 'YesNo3'),
+        ('yesno4', 'YesNo4', 'YesNo4'),
+        ('yesno2', 'YesNo2', 'YesNo2'),
+        ('yesno5', 'YesNo5', 'YesNo5'),
+    ],
+    'relativeage': [
+        ('yesno2', 'YesNo2', 'YesNo2'),
+        ('yesno1', 'YesNo1', 'YesNo1'),
+    ],
+    'collectionobject': [
+        ('collectionObjectType', 'Type', 'The type of object, such as a fish, mammal, mineral, rock, or meteorite.'),
+        ('cojo', 'Collection Object Group', 'Connects a Collection Object to its Collection Object Group'),
+    ],
+    'tectonicunit': [
+        ('guid', 'GUID', 'GUID'),
+        ('yesno1', 'YesNo1', 'YesNo1'),
+        ('tectonicUnitId', 'Tectonic Unit Id', 'Tectonic Unit Id'),
+        ('yesno2', 'YesNo2', 'YesNo2'),
+    ],
+    'tectonicunittreedefitem': [
+        ('createdbyagent', 'Created By Agent', 'Created By Agent'),
+        ('rankId', 'Rank Id', 'Rank Id'),
+    ]
+}
+
+MIGRATION_0023_FIELDS_BIS = {
+    'CollectionObjectGroup': ['guid', ' text3', 'decimal2', 'igsn', 'text2', 'collection', 'description', 'text1', 'cojo', 'decimal1', 'yesno3', 'integer3', 'yesno2', 'collectionObjectGroupId', 'integer2', 'yesno1', 'integer1', 'decimal3', ],
+
+    'collectionobjectgroupjoin' : ['yesno2', 'text1', 'yesno1', 'integer3', 'integer2', 'integer1', 'text3', 'yesno3', 'precedence', 'text2'],
+
+    'collectionobjectgrouptype' : ['collection'],
+
+    'collectionobjecttype': ['text3', 'collectionObjectTypeId', 'text2', 'text1', 'collection'],
+
+    'absoluteage': ['collectionDate', 'absoluteAgeId', 'date1', 'date2', 'yesno1', 'yesno2', 'agent1', 'number1', 'number2', 'collectionObject', 'absoluteAgeCitations', 'text1', 'text2'],
+
+    'paleocontext': ['yesNo1', 'text2', 'discipline', 'yesNo4', 'text2', 'yesNo2', 'number1', 'text3', 'number4', 'number5', 'yesNo3', 'text1', 'text5', 'number3', 'collectionObjects', 'text4', 'number2', 'chronosStratEnd', 'yesNo5'],
```

Comment:
Upon further consideration, I think we may want to avoid adjusting field visibility for Paleo Context, as it is already in use. We don't want to inadvertently change field captions or visibility in the schema, as this could create issues for users who have already customized those settings.